cap/modules/fixtures/schemas/cms.json

```diff
@@ -1200,6 +1200,7 @@
     "input_data": {
       "mc_sig_datasets": {
         "items": {
+          "ui:widget":"select",
```

Comment:
lets add for `primary` and `mc background` also..maybe check other schemas also

Reply:
Updated!